ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -571,6 +571,9 @@ extension ReactIterableAPI: IterableAuthDelegate {
                 DispatchQueue.main.async {
                     completion(self.passedAuthToken)
                 }
+            } else {
+                ITBInfo("authTokenRetrieval timed out")
+                completion(nil)
```

Comment:
The other `completion` call is made from the main queue, should we do the same here?

Reply:
AH yes it should be.